src/OrchardCore/OrchardCore.Workflows.Abstractions/Activities/Activity.cs

```diff
@@ -0,0 +1,135 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Localization;
+using Newtonsoft.Json.Linq;
+using OrchardCore.Entities;
+using OrchardCore.Workflows.Abstractions.Models;
+using OrchardCore.Workflows.Models;
+
+namespace OrchardCore.Workflows.Activities
+{
+    public abstract class Activity : Entity, IActivity
+    {
+        public abstract string Name { get; }
+        public abstract LocalizedString Category { get; }
+        public abstract LocalizedString Description { get; }
```

Comment:
If it's used only when displaying the list, we should remove it and let the Design views handle it.

Reply:
That's indeed the case, but I am indecisive with regards to whether we want to allow individual activities to have full control over their display, or use a shared shape that then uses these activity's properties to render (and still allow customization via alternates).  Thoughts?